frontend/components/CookieBanner.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import { css } from 'react-emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { sans, serif } from '@guardian/pasteup/fonts';
+import { Container } from '@guardian/guui';
+import TickIcon from '@guardian/pasteup/icons/tick.svg';
+import { getCookie, addCookie } from '../lib/cookie';
+
+const banner = css`
+    position: fixed;
+    bottom: 0;
+    background-color: ${palette.sport.dark};
```

Comment:
Use cases like this makes me regret calling it `sport` in the palette.

Reply:
ha true

Reply:
Gosh, I wonder if we should name these variables after the appropriate colours of the rainbow?

Reply:
But seriously, probably non-semantic names are best - e.g. like we do with the neutral colours.

Reply:
Hmm an argument for semantic names is that they're easier to reason about when say dynamically picking a palette colour dependent on an articles pillar. EG.

 ``
const myStyle = pillarName => css`
    color: ${palette[pillarName].pastel}
` ;
``

If we were just using numbers as we do for neutrals we'd need to be sure each pillar's palette uses the same number for the pastel colour, it seems easier to make a mistake and mismatch the colour to the wrong number than the wrong semantic name.